cloudmarker/util.py

```diff
@@ -32,6 +33,51 @@ def load_config(config_paths):
     return config
 
 
+def load_plugin(plugin_config):
+    """Construct an object with specified plugin class and parameters.
+
+    The plugin_config parameter must be a dictionary with the following
+    keys:
+
+      - 'plugin': The value for this key must be a string that
+        represents the fully qualified class name of the plugin. The
+        fully qualified class name is in the dotted notation, e.g.,
+        ``pkg.module.ClassName``.
+      - 'params': The value for this key must be a dict that represents
+        the parameters to be passed to the ``__init__`` functio of the
```

Comment:
`function` typo

Reply:
Updated the commit and force-pushed it. Thanks for noticing this.